docs/csharp/debugging.md

```diff
@@ -215,7 +215,7 @@ The runtime added support for applying changes while debugging on Linux/macOS on
 | Console | ✅ | Linux/macOS Only |
 | Test Projects | ✅ | Linux/macOS Only |
 | Class Library Projects | ✅ | Linux/macOS Only |
-| ASP.NET Core | ⚠* _Currently only supports changes on `.cs` files_ | Linux/macOS Only |
```

Comment:
I can't spot the change here 🤔

Reply:
Using a yellow emoji instead of black and white (and somewhat smaller) icon

Reply:
Oh, looks like it doesn't render for me
![CleanShot 2023-11-15 at 09 49 36@2x](https://github.com/microsoft/vscode-docs/assets/25163139/361864d6-2a17-456c-96dd-9959dc5ec0a4)

Reply:
You don't see the yellow emoji in the green update?
![image](https://github.com/microsoft/vscode-docs/assets/12818376/396ff8ff-775c-45ab-bbf9-ea2074723e14)

Reply:
Nope. Edge macOS

![CleanShot 2023-11-15 at 09 52 34@2x](https://github.com/microsoft/vscode-docs/assets/25163139/628c5d21-7f65-49f8-9c6b-4eca2502f77f)

Reply:
Are you reviewing on github.com or some other way, maybe GHPRI extension?

Reply:
Looks OK to me with the in-product diff view from GHPRI

![image](https://github.com/microsoft/vscode-docs/assets/12818376/1ba4575e-b2ae-44ae-9bb5-f74c37386086)

Reply:
On dot com. But I see the same behavior reviewing with GHPRI

![CleanShot 2023-11-15 at 09 53 57@2x](https://github.com/microsoft/vscode-docs/assets/25163139/ce7f0bdb-9a45-4092-a6db-45f2bc8d3b86)

Reply:
Could just be a font thing? Some fonts fall back to these basic emoji symbols IIRC

Reply:
I just grabbed https://emojipedia.org/warning. I wonder if it doesn't render correctly on all platforms. I'm on Windows 11. Is there a better place to grab the warning emoji?

Reply:
Weirdly selecting this via the native emoji picker..

![CleanShot 2023-11-15 at 09 59 41@2x](https://github.com/microsoft/vscode-docs/assets/25163139/13c4539c-e3c6-4a6c-a3ba-1942d1f876a1)

Results in... 
![CleanShot 2023-11-15 at 10 00 10@2x](https://github.com/microsoft/vscode-docs/assets/25163139/dc4e0a3d-1fa3-44c5-b115-90fb01faceb7)

But then it renders in the markdown preview:

⚠️

So maybe it will ultimately work just fine on the docs site.